app/controllers/questions_controller.rb

```diff
@@ -42,10 +43,19 @@ def destroy
     redirect_to questions_path
   end
 
+  def vote
+    @question.votes.create(user_id: current_user.id, status: params[:vote]) unless owner?(@question.user_id)
```

Comment:
Keep in mind sometimes people do Permitted Params:
Something like:

def permitted_params
  params.require(:object).permit(:foo)
end